test/cim/cim_creators.py

```diff
@@ -1108,6 +1109,20 @@ def create_loop(include_runtime: bool = True):
     )
 
 
+def create_lv_feeder(include_runtime: bool = True):
+    runtime = {
+        "normal_energizing_feeders": lists(builds(Feeder, **create_identified_object(include_runtime)), min_size=1, max_size=2),
+        "current_equipment": lists(builds(Equipment, **create_identified_object(include_runtime)), min_size=1, max_size=2)
```

Comment:
todo: This is building `Equipment` which is not a leaf class. I am not sure if/how this is even working. It should be using `sampled_equipment` like the `create_feeder` call.